tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Create extra directories relevant for systemd-networkd
+  file:
+    path: "{{ item | dirname }}"
+    state: directory
+  loop: "{{ systemd_network_copy_files | map(attribute='dest') | list + ['/etc/systemd/networkd.conf.d/', '/etc/systemd/resolved.conf.d/', '/etc/network/'] }}"
```

Comment:
Why `/etc/network`?

Reply:
Because at the end of the file, you add a dummy /etc/network/interfaces file there explaining the network is now systemd managed. That part fails (stops ansible play) if the directory does not exists. Maybe would be better create that dummy file only if the folder exists? Not sure...